apache.Dockerfile

```diff
@@ -10,12 +10,28 @@
 
 FROM docker.io/node:8.16.2 as builder
 
+
+RUN apt-get update \
+    && apt-get install -y git curl \
+    && apt-get -y clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN if [ "$(uname -m)" = "ppc64le" ]; then \
+     mkdir /tmp/phantomjs \
+     && curl -Ls "https://github.com/ibmsoe/phantomjs/releases/download/2.1.1/phantomjs-2.1.1-linux-ppc64.tar.bz2" | tar -xj --strip-components=1 -C /tmp/phantomjs \
+     && cd /tmp/phantomjs \
+     && mv bin/phantomjs /usr/local/bin \
+     && rm -rf /tmp/phantomjs; fi
+
 COPY package.json /dashboard/
 COPY yarn.lock /dashboard/
+
 WORKDIR /dashboard
 RUN yarn install --ignore-optional
```

Comment:
@bivasda1  Do we need
```
RUN if [ "$(uname -m)" = "ppc64le" ]; then export QT_QPA_PLATFORM=offscreen; fi \
    && yarn install --ignore-optional
```
 there?

Reply:
@atoulme It's not required for ppc64le arch.
Thanks